cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -586,6 +588,16 @@ def create_checkerKey(self):
                 "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
         return str(json_out[0]['key'])
 
+    def get_ceph_dashboard_link(self):
+        cmd_json = {"prefix": "mgr services", "format": "json"}
+        ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
+        # if there is an unsuccessful attempt,
+        if ret_val != 0 or len(json_out) == 0:
+            return None
```

Comment:
Do we want to return `None` on errors?

Reply:
What about `err_msg` where is it used?

Reply:
Intentionally we return none. Because, there are chances for the ceph dashboard is muted or not enabled. We do not want to handle or propagate the error and just want to ignore it without interrupting the further script execution.
The err_mag is also not used. I will void the err_msg by using '_'.